source/draw/gpu/GpuYuv420Semiplanar.ooc

```diff
@@ -85,5 +85,21 @@ GpuYuv420Semiplanar: class extends GpuImage {
 		this y fill(ColorRgba new(yuv y, 0, 0, 255))
 		this uv fill(ColorRgba new(yuv u, yuv v, 0, 255))
 	}
+	_toRgba: func (target: GpuImage) {
+		shader := this _context getYuvToRgba()
+		shader add("texture0", this y)
+		shader add("texture1", this uv)
+		DrawState new(target) setMap(shader) draw()
+	}
+	toRgb: func -> GpuImage {
+		target := this _context createRgb(this size)
+		this _toRgba(target)
+		target
+	}
+	toRgba: func -> GpuImage {
```

Comment:
We should rename `toRgba` for `GraphicBufferYuv420Semiplanar` to `asRgba` to prevent confusion. `as` = typecast, `to` = conversion